src/modules/folder/folder.repository.ts

```diff
@@ -459,4 +514,6 @@ export class SequelizeFolderRepository implements FolderRepository {
     return domain.toJSON();
   }
 }
+
+// QUESTION: Why is it exporting from here?
```

Comment:
Remove this comment. Remove the export if it should not be here. What do you think about this export @apsantiso ?

Reply:
I removed the FolderModel exports from folder.repository